geoprocessing-tests/geocoding.spec.ts

```diff
@@ -27,62 +30,68 @@ test.describe("Geocoding Tests", () => {
         }) => {
           console.log(`Running test: ${testCase.source} (${testCase.vintage})`);
 
-          // Append the API Key query param to the query URL. Pull this from your environment variables and default to 'demo' test key.
-          // console.log(process.env);
-
+          //Get .csv file name and stabish a connection to the desired .csv file for the lost of Domains to test
+          //NOTE: As of current version, the first Domain listed will be used as the base to be tested against.
+          const domainsCSV = parse(fs.readFileSync(path.join(__dirname, "../data/Domain_test2.csv")),{
+            //Insert desired .csv formatting parameters.
+          });
+          const domains = domainsCSV[0];
```

Comment:
Assert `Array<string>` type for domains so IDE knows what domains is. Without it, it infers `any` type.

So:

```js
          const domains: Array<string> = domainsCSV[0];
```